1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,24 @@
 function readingList(books) {
-  // Write your code here...
+  for (i = 0; i <= books.length; i++) {
+    let liElement = document.createElement("li");
+    let pElement = document.createElement("p");
+    let bookInfo = books[i].title + " by " + books[i].author;
+    pElement.textContent = bookInfo; // textContent only targets the text portion but leave any HTML alone, but if you go after html it will completely ruin HTML that you had before.
+    if (books[i].alreadyRead === true) {
+      liElement.classList.add("green");
+    } else {
+      liElement.classList.add("red");
+    }
+    liElement.appendChild(pElement);
+    let imgElement = document.createElement("img");
+    let bookImageUrl = books[i].bookCoverImage;
+    imgElement.src = bookImageUrl;
+    liElement.appendChild(imgElement);
+    let ulElement = document.querySelector("#reading-list");
+    ulElement.appendChild(liElement);
+    let divElement = document.querySelector("#content");
```

Comment:
I would suggest using a more specific selector for the ul and div elements. `#content` is very general